src/styles/blocks/_contacts.scss

```diff
@@ -0,0 +1,30 @@
+.contacts {
+  &__info {
+    margin: 0 0 24px;
+
+    &--last {
+      margin-bottom: 32px;
+    }
+  }
+
+  &__info-title {
+    margin: 0 0 8px;
+
+    font-size: 14px;
+    line-height: 27px;
+    color: #9fadc2;
+  }
+
+  &__info-link {
+    color: inherit;
+    text-decoration: none;
+  }
+
+  &__field {
+    margin: 0 0 16px;
+
+    &--last {
+      margin-bottom: 32px;
+    }
+  }
```

Comment:
Please, pay attention to my comments. You should add hover styles to this class to fix that thing from checklist:
4. Change text color on hover for phone, email and address